### DISCUSSION TITLE
Please where is the Aspire.Npgsql.EntityFrameworkCore.PostgreSQL v9 pacakges

### DISCUSSION THREAD

**Discussion by alfkonee** [microsoft/aspire#6736] [Q&A] [answered]
Hello pelase I'm trying to migrate to .Net 9 but it would see that the `Aspire.Npgsql.EntityFrameworkCore.PostgreSQL` packages are sill in RC when can we expect the 9.0.0 package to be  provided

@davidfowl

**Comment by davidfowl** [comment, marked as answer]
You can follow the issue here https://github.com/dotnet/aspire/issues/6720
